app/src/main/java/com/keylesspalace/tusky/adapter/NotificationsAdapter.java

```diff
@@ -377,46 +431,37 @@ void setAvatars(@Nullable String statusAvatarUrl, @Nullable String notificationA
         public void onClick(View v) {
             switch (v.getId()) {
                 case R.id.notification_container:
-                    if (listener != null) listener.onViewStatusForNotificationId(notificationId);
+                    if (notificationActionListener != null) notificationActionListener.onViewStatusForNotificationId(notificationId);
                     break;
-                case R.id.notification_top_bar:
-                    if (listener != null) listener.onViewAccount(accountId);
+                case R.id.notification_top_text:
+                    if (notificationActionListener != null) notificationActionListener.onViewAccount(accountId);
                     break;
             }
         }
 
-        private void setupContentAndSpoiler(boolean shouldShowContentIfSpoiler) {
+        private void setupContentAndSpoiler(boolean shouldShowContentIfSpoiler, final LinkListener listener) {
+            this.linkListener = listener;
+
             boolean hasSpoiler = !TextUtils.isEmpty(statusViewData.getSpoilerText());
-            CharSequence content;
             if (!shouldShowContentIfSpoiler && hasSpoiler) {
-                if (statusViewData.getMentions() != null &&
```

Comment:
Sorry, but I don't understand why you undid this. I've tried to match web version, there's even an issue somewhere asking for this behavior. Please, consider using this for the rest of the app.

Reply:
Ok, but why was it only done for notifications? I found it highly confusing that in notifications mentions are outside spoiler, but not in normal toots

Reply:
Yes, it was only done for notifications because I didn't have time to change it everywhere.